README.md

```diff
@@ -1,49 +1,60 @@
-jbi.vim
-=======
+## motoman-inform-vim-plugin
 
-Vim syntax file for Motoman INFORM III.
-----------------------------------------------------------------------
+## [Vim][1] syntax and filetype plugin for [Motoman][2] INFORM III.
 
 ### Installation
 
-Copy jbi.vim to `~/.vim/syntax/` or `$VIMRUNTIME/syntax/`.
-Use the command
+#### Installation with vim-plug:
 
-```vim
-    :set ft=jbi
-```
+Put this in your .vimrc:
 
-to enable syntax highlighting for jbi files.
+    call plug#begin('~/.vim/plugged')
+      Plug 'matthijsk/motoman-inform-vim-syntax'
+    call plug#end()
 
-### Automatic filetype detection
+For the first installation run:
 
-You might want to add the following lines to `~/.vim/filetype.vim` to enable
-automatic detection of the jbi filetype:
-    
-```vim
-    " Detect INFORM III filetype.
-    augroup filetypedetect
-      au BufRead,BufNewFile *.jbi		setfiletype jbi
-      au BufRead,BufNewFile *.JBI		setfiletype jbi
-    augroup END
-```
+    :PlugInstall
 
-### Navigating jobs
+Update every once in a while with:
 
-It's nice to be able to quickly navigate through the source code, especially
-if you make heavy use of `CALL JOB:<JOBNAME>` statements. When editing a C or
-C++ file with Vim the current file under the cursor can be opened using `gf`
-or `CTRL-W_CTRL-F`. To enable this for .jbi files add the following to your
-.vimrc:
+    :PlugUpdate
+
+#### Manual installation
+
+Extract and copy the folders 
+`/doc`, `/ftdetect`, `/ftplugin`, `/indent` and `/syntax` 
+into your 
+`~/.vim/` or `%USERPROFILE%\vimfiles\` 
+directory. 
+Overwrite jbi.\* files from older installation.
+
+Put the following in your .vimrc:
+
+    syntax on
+    filetype plugin on
+
+You may have to run
+
+    :helptags ~/.vim/doc/
 
-```vim
-    " Add suffixes to enable 'gf' commands on CALL JOB:<JOB> lines.
-    set suffixesadd+=.JBI,.jbi
+or
 
-    " Set path to jbi files. Replace this with the path to your actual working
-    " directory.
-    set path+=/path/to/jbi/files
-```
+    :helptags ~/vimfiles/doc/
+
+to use the help within Vim after installation.
+
+    :help krl
```

Comment:
What is `krl`?

Reply:
ups, copy&pasted from my Kuka Robot Language plugin :)

I don't see the necessity for a documentation, maybe apart from the path setting. Will delete that part.